lib/api/v3/queries/schemas/query_schema_representer.rb

```diff
@@ -283,6 +282,15 @@ def filter_instance_schemas_href
               api_v3_paths.query_filter_instance_schemas
             end
           end
+
+          private
+
+          def excluded_filters
+            # TODO: The RelatableFilter is not supported by the schema dependencies yet
+            [
+              ::Queries::WorkPackages::Filter::RelatableFilter
```

Comment:
alternatively we could define a method lookup in the filter itself, that is called from here?